common/data_refinery_common/models/models.py

```diff
@@ -366,6 +364,29 @@ def save(self, *args, **kwargs):
         self.last_modified = current_time
         return super(ComputationalResultAnnotation, self).save(*args, **kwargs)
 
+
+class Pipeline(models.Model):
+    "Pipeline that is associated with a series of ComputationalResult records."""
+
+    name = models.CharField(max_length=255)
+    steps = ArrayField(models.IntegerField(), default=[])
+
+    class Meta:
+        db_table = "pipelines"
+
+
+class Processor(models.Model):
+    """Processor associated with a certain ComputationalResult."""
+
+    name = models.CharField(max_length=255)
+    docker_img = models.CharField(max_length=255)
```

Comment:
We should use pythonic rather than C-like names here, so `docker_image` and `environment`, please.

It'd also be great if these could have a `__str__` method so that they display nicely in the Django admin interface.
Looks good otherwise!

Reply:
Thank you @Miserlou!